plugins/hanb_extended.py

```
@@ -0,0 +1,20 @@
+# Suppress the original hanb command
+_original_loaded = True
+
+
+@hook.command("hanb", autohelp=False, priority=1)
+def hanb(text: str):
+    """- Prints a random hanb (extended)"""
+    # Lazy import to avoid circular issues
+    from plugins.hanb_new import NEW_HANBS
+
+    # We need to get the original HANBS - import the module data directly
+    import importlib
```

Flake8 (7.3.0)

[error] 'importlib' imported but unused

(F401)